pnpm-lock.yaml

```diff
@@ -869,6 +920,20 @@ packages:
     transitivePeerDependencies:
       - supports-color
 
+  /@babel/helper-remap-async-to-generator/7.18.9_@babel+core@7.19.6:
```

Comment:
Looks like we have quite a few duplicates like this one. Looks like [there's a `pnpm-deduplicate`](https://www.npmjs.com/package/pnpm-deduplicate) (similar to the Yarn one) but I've never personally used it.